recipes-core/everest-config/files/01-migrate-configuration.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStartPre=/bin/sh /usr/bin/everest-migrate-config.sh
```

Comment:
I agree with @mhei's remarks. Thanks for fixing. But this line still needs to be adapted.
Otherwise LGTM, untested.

Reply:
Good catch! I missed this.